src/js/apps/check-ins/check-ins-main_app.js

```diff
@@ -0,0 +1,21 @@
+import RouterApp from 'js/base/routerapp';
+
+import CheckInApp from 'js/apps/check-ins/check-in/check-in_app';
+
+export default RouterApp.extend({
+  routerAppName: 'CheckInsApp',
+
+  childApps: {
+    checkIn: CheckInApp,
+  },
+
+  eventRoutes: {
+    'checkin': {
+      action: 'showCheckIn',
+      route: 'check-in/:id/patient/:id',
```

Comment:
I think maybe this should be flipped where `patient` is the root of the url so that it matches `patient/{id}/action/{id}` since those are similar relationships